apps/comps/components/competition-info.tsx

```diff
@@ -82,7 +83,7 @@ export const CompetitionInfo: React.FC<CompetitionInfoProps> = ({
         <CellTitle>Skills</CellTitle>
         <div className="flex flex-wrap gap-2">
           <span className="rounded border p-2 text-xs capitalize">
-            {competition.type}
+            {formatCompetitionType(competition.type)}
```

Comment:
fixes this on the comp, which is currently showing "Trading" instead of "Crypto Trading"
<img width="328" height="290" alt="Screenshot 2025-07-22 at 8 17 08 PM" src="https://github.com/user-attachments/assets/8185dadd-f0e9-411b-94dc-6d22d8aacb81" />